src/PrestaShopBundle/Resources/views/Admin/TwigTemplateForm/entity_search_input.html.twig

```diff
@@ -0,0 +1,84 @@
+{#**
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ *#}
+
+{% block entity_search_input_widget %}
+  {%- set attr = attr|merge({
+    'data-prototype-template': form_row(prototype),
+    'data-prototype-name': prototype.vars.name,
+    'data-prototype-image': prototype_image,
+    'data-prototype-value': prototype_value,
+    'data-prototype-display': prototype_display,
+    'data-mapping-value': mapping_value,
+    'data-mapping-image': mapping_image,
+    'data-mapping-display': mapping_display,
+    'data-remove-modal-title': remove_modal_title,
+    'data-remove-modal-message': remove_modal_message,
+    'data-remove-modal-apply': remove_modal_apply,
+    'data-remove-modal-cancel': remove_modal_cancel,
+    'data-remote-url': remote_url|raw,
```

Comment:
```suggestion
    'data-remote-url': remote_url|e('html_attr'),
```

You maybe need to escape not only this one?

Reply:
For now it meets the use case, the other fields are quite simple and have no special characters in theory We can always improve this later if it becomes blocking

Reply:
In theroy :trollface: 
What about the modal message for example, can be easily change by translations system.
But at least, I'm against using raw when it's not for rendering html

Reply:
That makes sense indeed, and you're right we can't be sure about the translated content I'll add it everywhere (I guess we should always use this `|e('html_attr')` for our data attributes)